userguide/system.rst

```diff
@@ -434,17 +434,16 @@ There are also other options available.
   in the :guilabel:`Active` column. Only alphanumeric characters,
   underscores, and dashes are allowed in the name.
 
-* **Scrub:** :guilabel:`Scrub Boot Pool` is used to perform a
```

Comment:
The text in red is what should be present in this branch. Revert this change, with the addition of changing boot device to |os-device|.